pages/blog/_slug.vue

```diff
@@ -144,6 +154,34 @@ export default {
       ],
     }
   },
+  jsonld() {
+    return {
+      '@context': 'https://schema.org',
+      '@type': 'Article',
+      mainEntityOfPage: {
+        '@type': 'WebPage',
+        '@id': `https://formester.com/blog/${this.article.slug}`,
```

Comment:
```suggestion
        '@id': `https://formester.com/blog/${this.article.slug}/`,
```

should have a trailing slash as a convention on this site